templates/calm/user_greeting.tmpl

```diff
@@ -0,0 +1,8 @@
+     <div id="info">
+       <h2>About</h2>
+       <dl>
+         <dd>You can customize this short message in the index.tmpl of this template in order to tell your visitors what they find in your repositories.</dd>
+         <dd>Visit <a href="http://www.websvn.info">www.websvn.info</a> for more information about WebSVN.</dd>
```

Comment:
Change url from `http://www.websvn.info` to `https://websvnphp.github.io/`

Reply:
This should be a separate PR:

```
$ grep -r www.websvn.info .
./languages/catalan.php:$lang["POWERED"] = "Funciona amb <a href=\"http://www.websvn.info/\">WebSVN</a>";
./languages/chinese-simplified.php:// $lang["POWERED"] = "Powered by <a href=\"http://www.websvn.info/\">WebSVN</a>";
./languages/chinese-traditional.php:$lang["POWERED"] = "採用 <a href=\"http://www.websvn.info/\">WebSVN</a>架設";
./languages/czech.php:$lang["POWERED"] = "Poháněno <a href=\"http://www.websvn.info/\">WebSVN</a>";
./languages/danish.php:$lang["POWERED"] = "Leveret af <a href=\"http://www.websvn.info/\">WebSVN</a>";
./languages/dutch.php:$lang["POWERED"] = "Aangedreven door <a href=\"http://www.websvn.info/\">WebSVN</a>";
./languages/english.php:$lang["POWERED"] = "Powered by <a href=\"http://www.websvn.info/\">WebSVN</a>";
./languages/finnish.php:$lang["POWERED"] = "Moottorina <a href=\"http://www.websvn.info/\">WebSVN</a>";
./languages/french.php:$lang["POWERED"] = "Propulsé par <a href=\"http://www.websvn.info/\">WebSVN</a>";
./languages/german.php:// $lang["POWERED"] = "Powered by <a href=\"http://www.websvn.info\">WebSVN</a>";
./languages/hebrew.php:$lang["POWERED"] = "פועל על ידי <a href=\"http://www.websvn.info/\">WebSVN</a>";
./languages/hindi.php:$lang["POWERED"] = "<a href=\"http://www.websvn.info/\">WebSVN</a> से संचालीत";
./languages/hungarian.php:// $lang["POWERED"] = "Powered by <a href=\"http://www.websvn.info/\">WebSVN</a>";
./languages/indonesian.php:$lang["POWERED"] = "Didukung oleh <a href=\"http://www.websvn.info/\">WebSVN</a>";
./languages/italian.php:// $lang["POWERED"] = "Powered by <a href=\"http://www.websvn.info/\">WebSVN</a>";
./languages/japanese.php:// $lang["POWERED"] = "Powered by <a href=\"http://www.websvn.info/\">WebSVN</a>";
./languages/korean.php:// $lang["POWERED"] = "Powered by <a href=\"http://www.websvn.info/\">WebSVN</a>";
./languages/macedonian.php:$lang["POWERED"] = "Работи на <a href=\"http://www.websvn.info/\">WebSVN</a>";
./languages/marathi.php:$lang["POWERED"] = "<a href=\"http://www.websvn.info/\">WebSVN</a> ने संचालीत";
./languages/norwegian.php:$lang["POWERED"] = "Kjører på <a href=\"http://www.websvn.info/\">WebSVN</a>";
./languages/polish.php:$lang["POWERED"] = "Obsługiwane przez <a href=\"http://www.websvn.info/\">WebSVN</a>";
./languages/portuguese-br.php:// $lang["POWERED"] = "Powered by <a href=\"http://www.websvn.info/\">WebSVN</a>";
./languages/portuguese.php:// $lang["POWERED"] = "Powered by <a href=\"http://www.websvn.info/\">WebSVN</a>";
./languages/russian.php:$lang["POWERED"] = "Работает на <a href=\"http://www.websvn.info/\">WebSVN</a>";
./languages/slovak.php:// $lang["POWERED"] = "Powered by <a href=\"http://www.websvn.info/\">WebSVN</a>";
./languages/slovenian.php:// $lang["POWERED"] = "Powered by <a href=\"http://www.websvn.info/\">WebSVN</a>";
./languages/spanish.php:// $lang["POWERED"] = "Powered by <a href=\"http://www.websvn.info/\">WebSVN</a>";
./languages/swedish.php:$lang["POWERED"] = "Körs på <a href=\"http://www.websvn.info/\">WebSVN</a>";
./languages/turkish.php:$lang["POWERED"] = "<a href=\"http://www.websvn.info/\">WebSVN</a> tarafından çalıştırılmaktadır";
./languages/ukrainian.php:$lang["POWERED"] = "Працює на <a href=\"http://www.websvn.info/\">WebSVN</a>";
./languages/uzbek.php:$lang["POWERED"] = "<a href=\"http://www.websvn.info/\">WebSVN</a> dasturi ishlatilmoqda";
./templates/calm/index.tmpl:       <dd>Visit <a href="http://www.websvn.info">www.websvn.info</a> for more information about WebSVN.</dd>
```

Reply:
@michael-o I agree.